### PR TITLE
add support for managing the initial buffer content.

### PR DESCRIPTION
**Pull request by UiP9AV6Y**
the option will modify the buffer content upon startup and works
in combination with `OptionShowCompletionAtStart()`.

somewhat related to #139

### CODE REVIEW

File: option.go [hunk 1]
```diff
@@ -37,6 +37,14 @@ func OptionPrefix(x string) Option {
 	}
 }
 
+// OptionInitialText to set the initial buffer text
+func OptionInitialText(x string) Option {
```
Comment (c-bata) [inline option.go]: I feel `OptionInitialBufferText` is clear and precise. What do you think?

File: _example/simple-echo/main.go [hunk 2]
```diff
@@ -19,6 +21,7 @@ func completer(in prompt.Document) []prompt.Suggest {
 func main() {
 	in := prompt.Input(">>> ", completer,
 		prompt.OptionTitle("sql-prompt"),
+		prompt.OptionInitialText(strings.Join(os.Args[1:], " ")),
```
Comment (c-bata) [inline _example/simple-echo/main.go]: Could you revert the changes of this example? I want to keep this example simple.

File: option.go (hunk 1 above)
Comment (UiP9AV6Y) [reply option.go]: yes, makes sense. i have renamed the function.

**Comment by c-bata** [comment]
Thank you for your contribution!
